docs/tutorials.md

```diff
@@ -212,3 +212,121 @@ curl -k --ipv4 https://02.pilosa.local:10502/index/sample-index/query -d 'Bitmap
 #### What's Next?
 
 Check out our [Administration Guide](https://www.pilosa.com/docs/latest/administration/) to learn more about making the most of your Pilosa cluster and [Configuration Documentation](https://www.pilosa.com/docs/latest/configuration/) to see the available options to configure Pilosa.
+
+
+### Integer Fields Values
+
+#### Introduction
+
+Pilosa can store integer values associated to the columns in an index, and those values are used to support range and aggregate queries. In this tutorial we will show how to set up integer fields, populate those fields with data, and query the fields. The example index we're going to create will represent fictional patients at a medical facility and various bits of information about those patients.
+
+First, create an index called `patients`:
+```
+curl localhost:10101/index/patients \
+     -X POST 
+```
+
+Next, create a frame in the `patients` index called `measurements` which will represent information gathered about each patient.
+```
+curl localhost:10101/index/patients/frame/measurements \
+     -X POST \
+     -d '{"options":{"rangeEnabled": true}}'
+```
+
+In addition to storing rows of bits, a frame can also contain fields that store integer values. The next step creates three fields (`age`, `weight`, `tcells`) in the `measurements` frame.
+```
+curl localhost:10101/index/patients/frame/measurements/field/age \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 120}'
+
+curl localhost:10101/index/patients/frame/measurements/field/weight \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 500}'
+
+curl localhost:10101/index/patients/frame/measurements/field/tcells \
+     -X POST \
+     -d '{"type": "int", "min": 0, "max": 2000}'
+```
+
+<div class="note">
+It's possible to create a frame with multiple fields in a single step. To do that, you just include a "fields" attribute to the frame options like the example below:
+</div>
+
+```
+curl localhost:10101/index/patients/frame/measurements \
+     -X POST \
+     -d '{"options":{
+              "rangeEnabled": true,
+              "fields": [
+                  {"name": "age", "type": "int", "min": 0, "max": 120},
+                  {"name": "weight", "type": "int", "min": 0, "max": 500},
+                  {"name": "tcells", "type": "int", "min": 0, "max": 2000}
+              ]
+         }}'
+```
```

Comment:
I think this might read a little more clearly by starting with this compact single-request form, and have a note below saying something to the effect of: "You can add fields to an existing frame by posting to the [Create Field endpoint](https://www.pilosa.com/docs/latest/api-reference/#create-field):"

This would be a little more terse and give them a link to the handler documentation, which they will probably find useful.